webapp/src/components/backstage/playbook_editor/controls.tsx

```diff
@@ -288,6 +302,30 @@ export const RunPlaybook = ({playbook}: ControlProps) => {
     );
 };
 
+export const JoinPlaybook = ({playbook: {id: playbookId}, refetch}: ControlProps & {refetch: () => void;}) => {
+    const {formatMessage} = useIntl();
+    const currentUser = useSelector(getCurrentUser);
+    const {join} = usePlaybookMembership(playbookId, currentUser.id);
+    const {setFollowing} = useEditorFollowersMeta(playbookId);
+
+    return (
+        <PrimaryButtonLarger
+            onClick={async () => {
+                join();
```

Comment:
Do we need to wait for the `join` to complete?
```suggestion
                await join();
```